apps/juxtaposition-ui/src/images.ts

```diff
@@ -0,0 +1,129 @@
+import { Buffer } from 'node:buffer';
+// @ts-expect-error Missing upstream types for this library
+import TGA from 'tga';
+// @ts-expect-error Missing upstream types for this library
+import imagePixels from 'image-pixels';
+import { inflate, deflate } from 'pako';
+import { PNG } from 'pngjs';
+import bmp from 'bmp-js';
+import sharp from 'sharp';
+import { logger } from '@/logger';
+
+/** Ingests a BMP-format painting and converts it to the usual TGA format.
+ * @param painting base64-encoded bmp image
+ * @returns base64-encoded zlib-compressed TGA
+ */
+export async function processBmpPainting(painting: string): Promise<string | null> {
```

Comment:
Just throwing this out there, one of the stated goals of this PR is to keep consistent naming. On the style side of this, one of the things I've tried to keep consistent across all of our services is casing. We following the [Go style of handling initialisms, acronyms](https://google.github.io/styleguide/go/decisions#initialisms), etc. where they all receive the same casing. So in this case it would be `processBMPPainting`

Not going to reject the PR or anything but I felt like this was a good time to mention this for moving forward

Reply:
The PR is ready for merge, but we'll keep it in mind for next time